src/controllers/expense.controller.js

```diff
@@ -0,0 +1,89 @@
+'use strict';
+
+const expenseService = require('./../serveces/expense.serveces');
+
+const get = (req, res) => {
```

Comment:
If you have a `getOne` function, indicate that you need to give a larger name value to this function so that the name immediately tells you why the difference